test/functions/bradc/arrayFormals/formalArrArgErrorLoc.chpl

```diff
@@ -0,0 +1,7 @@
+proc foo(X: [1..3] real) {
+  writeln("In foo!");
+}
+
+var A: [1..10] real;
+
+foo(A);
```

Comment:
Note to self: Stopped reviewing here. 

FYI Brad, I'll probably finish reviewing later tonight or tomorrow.